_posts/2020-02-06-pwa-cache-me-if-you-can.markdown

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "PWA Cache me if you can"
+date: 2020-02-06
+tags: PWA react
+author: Natalie Laing
+image: img/code.jpg
+---
+
+
+### What is a PWA anyway?
+
```

Comment:
Might be worth stating what 'PWA' stands for in here and some key characteristics that makes it "progressive"